test/model/update.test-d.ts

```diff
@@ -42,6 +45,9 @@ db.user.update({
     firstName: 'next',
     age: 24,
     country: db.country.create({ code: 'de' }),
+    lastName: null,
+    // @ts-expect-error Unable to update non-nullable values to null
+    updatedAt: null
```

Comment:
_This_ is gorgeous.

Reply:
I've realised from this comment that I need to add a check that initial values passed to `create` are also correctly typed 😄

Reply:
Great catch! 👍